StorageUtility/Windows/StorageUtility.cpp

```diff
@@ -203,6 +203,30 @@ eErrorCode GetStorageAdapterProperty( HANDLE Handle, sStorageAdapterProperty& Ad
     return( eErrorCode::None );
 }
 
+eErrorCode GetPhysicalDiskNumber(HANDLE Handle, U32& PhysicalDiskNumber)
```

Comment:
Ok aHai, in my opinion, we used DeviceIoControl to issue command to Drive a lot. I think it's "not good" ! A U32 value is ambiguous. How about a struct {PhysicalDiskNumber, DevicePath} ?